agent/typescript/src/cli.ts

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
```

Comment:
Instead of putting all of the files in agent/typescript/src, put them all in a new top level directory called /worker